css-size-adjust-1/Overview.bs

```diff
@@ -75,20 +82,24 @@ Values {#values}
   also accept the <a>CSS-wide keywords</a> keywords as their property value.
   For readability they have not been repeated explicitly.
 
-Default size adjustment {#default-adjustment}
-=============================================
+Size adjustment {#size-adjustment}
```

Comment:
I assume you shouldn't change the ID here, so old fragment links continue to work.